### PR TITLE
fix(curriculum): demistify cat ear triangle Cat Painting step 22

### LINKED ISSUE #53607: Request to add 2-3 steps to responsive web design: cat painting to demistify CSS triangles
### Is your feature request related to a problem? Please describe.

I was a little confused by the process to create cat ears. After mentioning it on 100 days of code channel (discord) I saw at least one other person hadn't understood it. So maybe more?

- In step 23 we start to draw the first cat ear <div class=”cat-left-ear”></div>
- We go straight to the CSS trick with thick borders and transparent sides
- At this point in the learners journey they are still not that familiar with the process of using <div></div> to create squares & rectangles and the different ways they can then manipulate those
- This made for a slightly frustrating few steps in an otherwise enjoyable exercise.
- (at first before searching for tutorials on this I thought it was utilising ‘layers’ somehow to reveal colours of the parent <div> - wrong lol).

### Describe the solution you'd like

I'd like to add 2-3 steps. I have drafted out a sample of what they could look like. Please see my notes: 
[Cat painting proposal](https://tulip-attack-b0f.notion.site/Exercise-Learn-Intermediate-CSS-by-Building-a-Cat-Painting-382da14c6cb04dd08ff4efa134b6ec54?pvs=4)

### Describe alternatives you've considered

I don't think any alternative is needed. It's just a judgement call whether to leave it as it is which would require some learners to do some additional research or to make modification to add a step or two. I think it's good to have learners do some side research, my worry is that some people will just skip it blindly following the instructions and not understand.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53607

image

image

image

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.height == '0px')
+```
+
+Your `.cat-right-ear` selector should have a `width` property set to `0`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.width === '0px')
+```
+
+Your `.cat-right-ear` selector should have a `background-color` property set to `white`. Don't forget to add a semi-colon.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md:35]:
```suggestion
Your `.cat-right-ear` selector should have a `background-color` property set to `white`. Don't forget to add a semicolon.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646ceb843412c74edee27a79.md [hunk 2]
```diff
@@ -7,34 +7,60 @@ dashedName: step-22
 
 # --description--
 
-You are going to make each ear look like a triangle.
+Now you will learn a CSS trick to draw triangles.
 
-Using a class selector, give the `.cat-left-ear` element a left and right border of `35px solid transparent` each. Also, set the bottom border to `70px solid #5e5e5e`.
+This will be used to draw ears on the cat.
+
+Using a class selector, give the `.cat-right-ear` element `height` and `width` properties set to `100px`. Set the `background-color` to white. Set left and right border to `35px solid blue`. Set top and bottom border to `35px solid red`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646ceb843412c74edee27a79.md:14]:
```suggestion
Using a class selector, give the `.cat-right-ear` element `height` and `width` properties set to `100px`. Set the `background-color` to white. Set the left and right borders to `35px solid blue`. Set the top and bottom borders to `35px solid red`.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8a8d9861319ee901a90.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+title: Step 24
+challengeType: 0
+dashedName: step-24
+---
+
+# --description--
+
+Remove the properties from the `.cat-right-ear` selector, also remove the `.cat-right-ear` selector.
+
+You are going to make each ear look like a triangle.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8a8d9861319ee901a90.md:12]:
```suggestion
Now you can begin working on your cat's ears.

Clean up your review code by removing the `.cat-right-ear` selector and all of its properties.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+title: Step 23
+challengeType: 0
+dashedName: step-23
+---
+
+# --description--
+
+Notice that you now have a white square with thick borders. The borders have diagonal edges which can be used to create triangles.
+
+Within the same class selector adjust `height` and `width` to `0`. Set the left, right and top border to `transparent`.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md:12]:
```suggestion
Notice that you now have a white square with thick borders. The borders have diagonal edges which can be used to create triangles.

Within the same class selector adjust `height` and `width` to `0`. Set the left, right and top border to `transparent`.

Remove the `background-color` property, and you should see a triangle.
```

Let's actually have them complete the triangle.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.borderTop === '35px solid transparent')
+```
+
+
+Your `.cat-right-ear` selector should have a `border-bottom` property set to `35px solid red`. Don't forget to add a semi-colon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.borderBottom === '35px solid red')
+```
+
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md:65]:
````suggestion

Your `.cat-right-ear` selector should not have a `background-color` property.

```js
assert(!new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.backgroundColor)
```

````

And test for it.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8a8d9861319ee901a90.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+  height: 180px;
+  border: 1px solid #000;
+  border-radius: 46%;
+}
+
+--fcc-editable-region--
+.cat-right-ear {
+  height: 0;
+  width: 0;
+  background-color: white;
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8a8d9861319ee901a90.md:147]:
```suggestion
```

This'll have to get tossed from the seed.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8a8d9861319ee901a90.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+```js
+assert(!new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.width)
+```
+
+Your `.cat-right-ear` selector should not have a `background-color` property.
+
+```js
+assert(!new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.backgroundColor)
+```
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8a8d9861319ee901a90.md:35]:
```suggestion
```

And then we don't gotta test for it.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.width === '0px')
+```
+
+Your `.cat-right-ear` selector should have a `background-color` property set to `white`. Don't forget to add a semicolon.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.cat-right-ear')?.backgroundColor === 'white')
+```
+
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/6676a8b01e56ec1a1e07c202.md:42]:
```suggestion
```


My fault, sorry

**Review by naomi-lgbt — APPROVED** [review]
Awesome! Thanks for all your work on this!

**Note to reviewers:** Once this is approved a second time, I'll run + push the i18n sync and merge

**Review by Sembauke — APPROVED** [review]

**Comment by ilenia-magoni** [comment]
@Sembauke you did not let Naomi do the i18n sync, and now the `main` branch will fail

**Comment by Sembauke** [comment]
I misunderstood, sorry about that.

**Pull request merged**